analyzer/analyzer.go

```diff
@@ -16,6 +16,15 @@ import (
 	"golang.org/x/tools/go/analysis"
 )
 
+const (
+	intConversionChecker string = "int-conversion"
+	errErrorChecker      string = "err-error"
+	errorfChecker        string = "errorf"
+	sprintf1Checker      string = "sprintf1"
+	fiximportsChecker    string = "fiximports"
+	strconcatChecker     string = "strconcat"
```

Comment:
Please give them a common prefix, not suffix

For code completion plus clarity

Reply:
Thanks @mmorel-35 for doing the changes.

@alexandear do you know any linters about this?

Maybe it could be part of revive. I find this pattern so frequently in code.

Of course, this linter would be opinionated and might cause problem for exported constants

What do you think ?

We could move the discussion to revive repository maybe, but I would like to get feedbacks from you guys, before going further.